src/Sylius/Behat/Context/Api/Shop/CheckoutContext.php

```diff
@@ -716,11 +722,14 @@ public function thereShouldBeNoTaxesCharged(): void
     }
 
     /**
-     * @Then my order's locale should be :localeCode
+     * @Then my order's locale should be :locale
      */
-    public function myOrderLocaleShouldBe(string $localeCode): void
+    public function myOrderLocaleShouldBe(LocaleInterface $locale): void
     {
-        Assert::same($this->responseChecker->getValue($this->ordersClient->getLastResponse(), 'localeCode'), $localeCode);
+        Assert::same($this
+            ->responseChecker
+            ->getValue($this->ordersClient->getLastResponse(), 'localeCode'), $locale->getCode()
+        );
```

Comment:
Why did you change it?

Reply:
In @ui tests was using locale name, but in @api we need localeCode, It worked before, because test for `en_US` working properly for both, but to use fr_FR created problems